src/content/structured/get-started/react.mdx

```diff
@@ -103,3 +103,35 @@ module.exports = withTM(nextConfig);
 The `@ukic/react` package will need to be transformed before you can use these components in Jest tests.
 
 Add a `transformIgnorePatterns` field with the value `["/node_modules/(?!@ukic/react)"]` to your Jest config.
+
+## Working with slotted SVGs
+
+### Step one
+
+To slot an SVG into one of our React components, import the `SlottedSVG` component from `@ukic/react` into your React files. This will prevent an error stating that property 'slot' doesn't exist.
+
+```jsx
+import { SlottedSVG } from "@ukic/react";
+```
+
+### Step two
+
+Then update your code to replace any instances of `<svg>` with `<SlottedSVG>`.
+
+As an example, when using IcTopNavigation, if your existing code looks like this:
+
+```jsx
+<IcTopNavigation appTitle="App title">
+  <svg slot="app-icon" path={path} />
+</IcTopNavigation>
+```
+
+<br />
+
+Then it should be replaced with:
+
+```jsx
+<IcTopNavigation appTitle="App title">
+  <SlottedSVG slot="app-icon" path={path} />
+</IcTopNavigation>
+```
```

Comment:
I think here we can do something like this as it's a long page anyway:

~~As an example, when using IcTopNavigation, if your existing code looks like this:~~

```jsx
// Before
<IcTopNavigation appTitle="App title">
  <svg slot="app-icon" path={path} />
</IcTopNavigation>

// After
<IcTopNavigation appTitle="App title">
  <SlottedSVG slot="app-icon" path={path} />
</IcTopNavigation>
```